meson.build

```diff
@@ -795,10 +795,6 @@ conf.set_quoted('SYSTEMD_DEFAULT_LOCALE', default_locale)
 
 conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
 
-service_watchdog = get_option('service-watchdog')
-substs.set('SERVICE_WATCHDOG',
-           service_watchdog == '' ? '' : 'WatchdogSec=' + service_watchdog)
```

Comment:
as far as I can tell, this use of ? is what totally broke meson, according to https://github.com/mesonbuild/meson/issues/5003

This is probably fixed in later versions of meson, but if it can be done a different way that doesn't break meson 0.49 it would be easier :)